frontend/lib/ui/privacy-info-modal.tsx

```diff
@@ -40,29 +70,8 @@ export function PrivacyInfoModal(props: {
               tenants rights mission. The Privacy Policy contains information
               regarding what data we collect, how we use it, and the choices you
               have regarding your personal information. If you’d like to read{" "}
-              more, please review our full{" "}
-              <OutboundLink
-                href={
-                  props.isForNorentSite
-                    ? addNorentSuffixToUrl(DEFAULT_PRIVACY_POLICY_URL)
-                    : DEFAULT_PRIVACY_POLICY_URL
-                }
-                target="_blank"
-              >
-                Privacy Policy
-              </OutboundLink>{" "}
-              and{" "}
-              <OutboundLink
-                href={
-                  props.isForNorentSite
-                    ? addNorentSuffixToUrl(DEFAULT_TERMS_OF_USE_URL)
-                    : DEFAULT_TERMS_OF_USE_URL
-                }
-                target="_blank"
-              >
-                Terms of Use
-              </OutboundLink>
-              .
+              more, please review our full <PrivacyPolicyLink />
```

Comment:
Seems you are missing a space after the `<PrivacyPolicyLink />`.

Reply:
ah thanks, fixed!